clusterman/autoscaler/autoscaler.py

```diff
@@ -296,51 +277,23 @@ def _compute_target_capacity(self, resource_request: SignalResourceRequest) -> f
             )
             return current_target_capacity
 
-        # If we get here, everything is non-zero and we can use the "normal" logic to determine scaling
```

Comment:
All the logic is quite complicated to reason about, how can we be sure that we're not introducing new bugs here by replacing it?